doc/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -50,7 +51,8 @@
 
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, "motion_planning_tutorial");
+  std::string name_ = "motion_planning_tutorial";
```

Comment:
- you should not have trailing underscores for a local variable
- add const for the string
- better variable name e.g. node_name